developer/src/tike/project/Keyman.Developer.UI.Project.ProjectUI.pas

```diff
@@ -49,20 +53,53 @@ function IsGlobalProjectUIReady: Boolean;
 procedure FreeGlobalProjectUI;
 begin
   FreeAndNil(FGlobalProject);
+  TikeMultiProcess.CloseProject;
 end;
 
-function LoadGlobalProjectUI(pt: TProjectType; AFilename: string): TProjectUI;
+function CreateTempGlobalProjectUI(pt: TProjectType): TProjectUI;
+var
+  kpj: TProject;
+  AFilename: string;
 begin
   Assert(not Assigned(FGlobalProject));
-  if DirectoryExists(AFilename) then
-  begin
-    // Load a directory-based project
-    if AFilename.EndsWith('\') then
-      AFilename := AFilename.Substring(0, AFilename.Length-1);
-    AFilename := AFilename + '\' + ExtractFileName(AFilename) + '.kpj';
+
+  AFileName := KGetTempFileName('.kpj');
+  System.SysUtils.DeleteFile(AFileName);
+
+  kpj := TProject.Create(pt, AFilename, False);
+  try
+    kpj.Options.Version := pv10;
```

Comment:
Why isn't this pv20 like in [KeyboardProjectTemplate.pas](#10114/files#diff-b842a61d77902112e7b345c4133faafed6b8a0f341bc1957d8c3d6e87b8da264)?

Reply:
We use a pv10 format here because it allows us to add arbitrary files. pv20 _only_ allows files in the project path.